src/Microsoft.Extensions.DependencyModel/DependencyContextPaths.cs

```diff
@@ -7,32 +12,49 @@ namespace Microsoft.Extensions.DependencyModel
     internal class DependencyContextPaths
     {
         private static readonly string DepsFilesProperty = "APP_CONTEXT_DEPS_FILES";
```

Comment:
The reason is because there is a place where we use the SharedRuntime property to find the directory of the framework. And I didn't like that we assume a specific index was the SharedFx deps file. There is an AppContext parameter for the SharedFx deps file, so I felt it was correct to use it instead. Now the only order we assume is that the first deps file is the entry assembly's.

But I can change it so the SharedRuntime property is populated from the FX_Deps parameter. And in the loader, we don't use the SharedRuntime property.  Instead use the full string[] that was passed by APP_CONTEXT_DEPS_FILES in that order.

The SharedFx deps file would be in both, assuming the host passed it in both.

Thoughts?